index.html

```diff
@@ -724,8 +731,31 @@ <h2>Checkbox</h2>
 				</ts-button-group>
 			</ts-aside>
 			<ts-cover id="my-cover"></ts-cover>
+			<ts-modal id="big-modal" data-title="Available Apps">
+				<div id="big-content" slot="main"></div>
+				<div slot="footer">
+					<ts-button onclick="closeBigModal()">Close!</ts-button>
+				</div>
+			</ts-modal>
+			<ts-modal id="medium-modal" data-title="Available Apps" data-size="medium">
+				<ts-note slot="note">
+					Some important note about this information.
+				</ts-note>
+				<div id="medium-content" slot="main"></div>
+				<div slot="footer">
+					<ts-button onclick="closeModal()">Close!</ts-button>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a 'onKeyPress' attribute to this <ts-button> tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AMouseEventWithoutKeyboardEquivalentCheck)